VectorRumble.Shared/Screens/AboutMenuScreen.cs

```diff
@@ -92,7 +104,11 @@ private void GameEnhancementsMenuEntrySelected(object sender, EventArgs e)
         /// </summary>
         private void GithubRepoMenuEntrySelected(object sender, EventArgs e)
         {
-            System.Diagnostics.Process.Start("https://github.com/infinitespace-studios/VectorRumble");
+#if WINDOWS_UAP
+            ProcessStart(MonoGameUri);
+#else
+            System.Diagnostics.Process.Start(GitHubUri);

```

Comment:
just a suggestion. Move the Process.Start into the new `ProcessStart` method, and then call it here. We can remove a bunch of `#if` sections then :)

Reply:
Good call.  Bloody brain isn't working :(